app/models/results/csv/answer_query.rb

```diff
@@ -65,7 +65,7 @@ def order
             group1_rank NULLS FIRST,
             group1_inst_num NULLS FIRST,
             qings.rank,
-            answers.rank,
+            answers.old_rank,
```

Comment:
Dependency on `rank` is removed in develop. You can just `git co --theirs` when you rebase.